test/core/slice/slice_test.cc

```diff
@@ -243,7 +243,13 @@ static void test_slice_interning(void) {
   grpc_init();
   grpc_slice src1 = grpc_slice_from_copied_string("hello123456789123456789");
   grpc_slice src2 = grpc_slice_from_copied_string("hello123456789123456789");
+
+  // Do not remove the log line. It actually supresses a compiler bug in windows
+  // bazel opt build. See https://github.com/grpc/grpc/issues/20519
```

Comment:
Is there a way to loudly mark the test as "skipped", rather than using this workaround? Very interesting finds though.

Reply:
We could just skip the test and file an issue to followup. I'm still trying to find the actual MSVC bug listing to make 100% sure this is actually a compiler bug, but it's very time-consuming as I don't have much to go off of.

So far:
- I've been able to reproduce under bazel with VS2015, VS2017 and VS2019
- when I build with cmake (Release build), I'm unable to reproduce -  which is pretty strange, but I haven't yet compared the exact compiler flags being used (but I checked they both run with /O2)